test-suite/js/src/test/scala/org/scalajs/testsuite/javalib/lang/SystemJSTest.scala

```diff
@@ -57,4 +59,58 @@ class SystemJSTest {
       assertEquals(x1FirstHash, System.identityHashCode(x1))
     }
   }
+
+  @Test def systemProperties(): Unit = {
+    def get(key: String): String = java.lang.System.getProperty(key)
+
+    // Defined in System.scala
+    assertEquals("1.8", get("java.version"))
```

Comment:
In the same commit as those, there should be one hard-coded one `-D` property in the `javaOptions` of the test-suite to test that the logic in `javaSysProps2JSLib` works.